### ISSUE TITLE
Сделать доступным просмотр сверхдолгих роликов

### ISSUE THREAD

**Issue by N-point-N** [N-point-N/ricktube#1]

**Comment by yulavis** [comment]
Не логинишься. Смотришь с помощью сервиса восьмичасовой летсплей с ютуба.  Говоришь себе: "Хм, полезная штука, надо подписку оформить." Регистрируешься, платишь. заливаешь ссылку на тот же летсплей, чтобы досмотреть... "Видео слишком длинное".  Л - логика.....

**Comment by N-point-N** [comment]
Да, есть такая ситуация сейчас, связано с ограничением по размеру видео, чтоб не забили канал общий роликами: играю в майнкрафт 130 часов :)

Лимит будет снят после реализации мгновенного просмотра без ожидания полной загрузки ролика
